src/components/Spinner.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import styled, { keyframes } from 'styled-components';
+import { IconLoaderOutline, BaseIconProps } from '@taskany/icons';
+
+const rotate = keyframes`
+    100% {
+        transform: rotate(360deg);
```

Comment:
i think end rotating value must be equals `359deg` because `360deg === 0deg` that may cause freezing animation on longest durations